metron-platform/metron-elasticsearch/src/test/java/org/apache/metron/elasticsearch/integration/ElasticsearchIndexingIntegrationTest.java

```diff
@@ -102,11 +102,16 @@ public void setAdditionalProperties(Properties topologyProperties) {
     topologyProperties.setProperty("es.clustername", "metron");
     topologyProperties.setProperty("es.port", "9300");
     topologyProperties.setProperty("es.ip", "localhost");
-    topologyProperties.setProperty("indexing.writer.class.name", "org.apache.metron.elasticsearch.writer.ElasticsearchWriter");
+    topologyProperties.setProperty("indexing_writer_class_name", "org.apache.metron.elasticsearch.writer.ElasticsearchWriter");
   }
 
   @Override
   public String cleanField(String field) {
     return field;
   }
```

Comment:
I think it's a valid concern.  I'm also not sure what the ideal alternative is.  At this point I feel like Metron has succumbed to being tightly coupled with Ambari.